buildSrc/build.gradle

```diff
@@ -10,6 +10,7 @@ repositories {
 
 dependencies {
   implementation 'com.android.tools.build:gradle:8.1.2'
+  implementation 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.8.22'
```

Comment:
without this, we get many strange errors in Compose compilation. Thanks to https://stackoverflow.com/questions/75995893/jetpack-compose-most-basic-buildsrc-build-error-illegalstateexception-coul